src/AppBundle/Resources/views/Pdf/delivery.html.twig

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
```

Comment:
I think to do clean invoices we will need the settings for admin/coop details (adress, siret, aso.).